JetNews/app/src/main/java/com/example/jetnews/ui/home/HomeScreen.kt

```diff
@@ -166,15 +170,37 @@ fun HomeScreen(
         topBar = {
             val title = stringResource(id = R.string.app_name)
             InsetAwareTopAppBar(
-                title = { Text(text = title) },
+                title = {
+                    Image(
+                        painter = painterResource(R.drawable.ic_jetnews_wordmark),
+                        contentDescription = title,
+                        colorFilter = ColorFilter.tint(MaterialTheme.colors.onBackground),
+                        modifier = Modifier
+                            .fillMaxSize()
+                            .padding(bottom = 4.dp, top = 10.dp)
+                    )
+                },
                 navigationIcon = {
                     IconButton(onClick = { coroutineScope.launch { openDrawer() } }) {
-                        Icon(
+                        Image(
```

Comment:
You can use an Icon here and set the tint

Reply:
Done

Reply:
I still see `Image`?

Reply:
+1, please replace with icon

From IconButton:
> content should typically be an Icon, using an icon from androidx.compose.material.icons.Icons. If using a custom icon, note that the typical size for the internal icon is 24 x 24 dp.

Reply:
I just made the change